ecosystem/indexer-server/docker-bake.hcl

```diff
@@ -0,0 +1,33 @@
+# This is a docker bake file in HCL syntax.
```

Comment:
random question - why is this path called `indexer-server` ? Can't we just call it `indexer` ? Seems like we call it just indexer everywhere else incl. the helm chart (and helm chart path).

Reply:
so the `indexer` tails the chain and writes to a database, and the `indexer-server` is an http server that serves requests on top of the DB

Reply:
I see. Let's call it `indexer-api` then. The other one probably should be called `indexer-ingester` or `indexer-builder` or something like that.